test/common/json/gen_excluded_unicodes.cc

```diff
@@ -0,0 +1,127 @@
+#include "source/common/protobuf/utility.h"
+
+#include "test/common/json/utf8.h"
+
+#include "absl/strings/str_format.h"
+
+namespace Envoy {
+namespace Json {
+namespace {
+
+// Collects unicode values that cannot be handled by the protobuf json encoder.
+// This is not needed for correct operation of the json sanitizer, but it is
```

Comment:
Nit: s/json/JSON/ here and elsewhere it is used as acronym. Similarly, s/unicode/Unicode/